frontend/src/ScoreBox.js

```diff
@@ -244,15 +243,16 @@ class ScoreBox extends Component {
             }
             return a ? 1 : -1
           },
-          Cell: props => {
+          Cell: ({ column, original: { key }, value = {} }) => {
             const classes = []
-            const value = props.value
+            const { average, recent } = Object.keys(value).length
+            const val = value
```

Comment:
You got these two backwards. Object.keys(value).length is a number, so you can't destructure from it. Please read over your code to make sure the types check out before committing :^)